llvm/test/Transforms/SampleProfile/overflow.ll

```diff
@@ -0,0 +1,72 @@
+; Checks that we are able to handle overflowing counters correctly.
+
+; RUN: opt < %s -passes=sample-profile -sample-profile-file=%S/Inputs/overflow.proftext | opt -passes='print<branch-prob>' -disable-output 2>&1 | FileCheck %s
```

Comment:
Can you try using `utils/update_analyze_test_checks.py` and auto-generate the `CHECK:` lines that makes for more robust tests? We haven't mass-edited older tests for this style, but it should be used for newly added tests.

You can find an example in `test/Analysis/BranchProbabilityInfo/basic.ll`.